imports/api/collectionsCount/server/hmisCounts.js

```diff
@@ -5,6 +5,15 @@ class HmisCounts {
     this.userId = userId;
   }
 
+  getClientsCount() {
+    const hc = HmisClient.create(this.userId);
+    const schemas = ['v2017', 'v2016', 'v2015', 'v2014'];
+    return schemas.reduce(
+      (count, schema) => count + hc.api('client').getClients(schema).length,
```

Comment:
Maybe we should send clients with limit=1 and look for pagination.total data

Reply:
+1